src/main/java/org/folio/dcb/service/TransactionService.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.dcb.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.domain.entity.Transactions;
+import org.folio.dcb.domain.dto.TransactionStatus;
+import org.folio.dcb.repository.TransactionsRepository;
+import org.folio.spring.exception.NotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+@Log4j2
+@RequiredArgsConstructor
+public class TransactionService {
+
+  private final TransactionsRepository transactionsRepository;
+
+  public TransactionStatus getTransactionStatusById(UUID transactionId) {
+    Transactions transaction = getTransactionOrThrow(transactionId);
+    TransactionStatus ts = new TransactionStatus();
+    ts.setStatus(transaction.getStatus());
+    ts.setMessage("message");
```

Comment:
Do we need to put message like this? i am not sure . I think there should be some meaningful String.
@Vignesh-kalyanasundaram Please suggest.

Reply:
Hi @SinghAdes , I am also thinking that we need to have some meaningful string in the message. But I am not aware of the use of the message field. So we can check with kalibek.

Reply:
Why do we need a message in the response ?